CHANGELOG.md

```diff
@@ -5,9 +5,11 @@ SPDX-License-Identifier: GPL-3.0-or-later
 -->
 
 # 0.5.4 (unreleased)
+- Add support for SP scalars and determine label and scalar size from env variable [PR #120](https://github.com/hpsim/OGL/pull/120) 
+- Notify user of a unsupported executor argument [PR #118](https://github.com/hpsim/OGL/pull/118) 
 - Notify user of a unsupported executor argument [PR #]
-- Add a list of all OGL authors [PR #113](https://github.com/hpsim/OGL/pull/113)
-- Replace #ifdef include guard with #pragma once [PR #113](https://github.com/hpsim/OGL/pull/113)
+- Add a list of all OGL authors [PR #126](https://github.com/hpsim/OGL/pull/126)
+- Replace #ifdef include guard with #pragma once [PR #126](https://github.com/hpsim/OGL/pull/126)
 - Switch to SPDX headers, fix formatting issue[PR #112](https://github.com/hpsim/OGL/pull/112)
```

Comment:
Switch to SPDX headers, fix formatting issue[PR #122](https://github.com/hpsim/OGL/pull/122)